src/ngx_http_lua_ssl_certby.c

```diff
@@ -952,6 +952,254 @@ ngx_http_lua_ffi_priv_key_pem_to_der(const u_char *pem, size_t pem_len,
 }
 
 
+void *
+ngx_http_lua_ffi_cert_pem_to_cdata(const u_char *pem, size_t pem_len,
```

Comment:
Maybe `ngx_http_lua_ffi_parse_pem_cert` is a better name? I don't really want implementation details like `cdata` be part of the API name :)